src/com/teamcautionrobotics/robot2018/Intake.java

```diff
@@ -0,0 +1,45 @@
+package com.teamcautionrobotics.robot2018;
+
+import edu.wpi.first.wpilibj.VictorSP;
+
+public class Intake {
+
+    private VictorSP intake1;
+    private VictorSP intake2;
+
+    public Intake(int motorPort1, int motorPort2) {
+
+        intake1 = new VictorSP(motorPort1);
+        intake2 = new VictorSP(motorPort2);
+
+    }
+
+    public void out(double power) {
```

Comment:
Same about combining  `void out(double power)` and `void in(double power)` as with the `Climb` class.